docs/install.md

```diff
@@ -589,18 +592,26 @@ A login prompt will appear. Use the initial credentials:
     sudo nmap -sn 192.168.1.0/24 #For scanning IP address
     ```
 
-    Alternatively you may also determine the IP address in the DHCP status page of your DHCP server (often included in a router).
+    Alternatively you may also determine the IP address in the DHCP status page of your DHCP server (often included in a router): Search for the hostname **dietpi** and obtain the IP address.
 
     **Connect to DietPi via SSH**
 
-    - A popular SSH Client for Windows is PUTTY. You can download putty from [here](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html). Enter in the `Host Name` field the IP address found during the scanning, select `SSH` and then click on _Open_ button.
+    - A popular SSH Client for Windows is PuTTY. You can download PuTTY from [here](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html). Enter in the `Host Name` field the IP address found during the scanning, select `SSH` and then click on _Open_ button.  
+    Depending on your DHCP configuration, also just `dietpi` may be sufficient as the hostname.  
+    Sometimes it needs to be followed by your router's domain (e.g. `dietpi.fritz.box`).  
     ![DietPi-SSH](assets/images/dietpi-ssh.jpg)
 
     - Most Linux distributions come packaged with an ssh client. Type in your Terminal next command (replace the sample IP address `192.168.1.20` with the one found via scanning the network):  
     ```
     ssh root@192.168.1.20
     ```
 
+        resp.
+
+        ```
+        ssh root@dietpi
```

Comment:
`root@DietPi`, as above.

Reply:
ssh does not use case-sensitivity.
Not changed for more easy entry (instead of `ssh root@DietPi`).

Reply:
Have you tested if it works? I mean the systems /etc/hostname can not even automatically be used to access the machine from the LAN, as long as the server does not broadcast it (via avahi-daemon e.g.) or the router catches it via DHCP. But if it does, at least in my router I see test DietPi systems listed with "DietPi" hostnames.

Actually, to avoid inconsistencies and in case confusions, we should switch to lower case "dietpi" for the hostname. At least for WAN hostnames it's definitely true that they're not case-sensitive.

Reply:
Checked ssh with PuTTY: no case-sensitivity
Checked X11 with xrdp: case-sensitive